src/app/protected/error.jsx

```diff
@@ -0,0 +1,20 @@
+"use client";
+import { signIn } from "next-auth/react";
+
+const protectedError = () => {
+  return (
+    <>
+      <p>401! You are not authorized to view this page!</p>
+      <button
+        onClick={(e) => {
+          e.preventDefault();
```

Comment:
pretty sure if u do 
```js
() => signIn("google")
```

it will work just as well, also that callbackURL is slightly risky cause we dont know what page we will be redirecting to